sql/migrations/schema-postgresql-move-to-mirrors.sql

```diff
@@ -0,0 +1,98 @@
+DROP TABLE IF EXISTS filemetadata, mirrors CASCADE;
+DROP MATERIALIZED VIEW IF EXISTS filemetadata_mirror_count;
+
+CREATE TABLE filemetadata
+(
+    id integer GENERATED ALWAYS AS IDENTITY,
+    path character varying NOT NULL,
+    path_hash bytea GENERATED ALWAYS AS (digest((path)::text, 'sha256'::text)) STORED,
+    mtime timestamp with time zone,
```

Comment:
the migration code already takes care of the int to timestamp conversion.

Reply:
There are other places in code, but it looks I was able to cover them in #52 . Let's hope that everything is covered since it is hard to pay attention on every detail with such many changes,